scripts/components/RtaForm.jsx

```diff
@@ -86,9 +74,17 @@ var style = {
     display: 'flex',
     flexFlow: 'row wrap',
   },
-  col: {
-    width: '50%',
-  }
+  col70: {
+    width: '70%',
+    padding: '0.313em',
+  },
+  col30: {
+    width: '30%',
+    padding: '0.313em',
+  },
+  card: {
+    marginBottom: '1.250em',
+  },
 };
 
-module.exports = RtaForm;
+module.exports = MuiContextified(RtaForm);
```

Comment:
Can you make sure there is a newline at the end of your file. See how github is showing a special icon to indicate this is likely a problem. Anyways, no newline at the end of files can cause issues for certain software. There should be a setting in your text editor to automatically add a newline at the end of any file on save.